pkg/query-service/rules/base_rule.go

```diff
@@ -482,6 +526,10 @@ func (r *BaseRule) shouldAlert(series v3.Series) (Sample, bool) {
 			if sum != r.targetVal() {
 				shouldAlert = true
 			}
+		} else if r.compareOp() == ValueOutsideBounds {
+			if math.Abs(sum) >= r.targetVal() {
```

Comment:
Why is valueOutsideBounds using just `>=` ?

Reply:
The `math.Abs` is playing the key role here. For example, if the target is 3 std dev away without specifying direction such as `above` or `below` then they should be alerted when it's away from the positive side or negative side.

Reply:
Got it